packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart

```diff
@@ -433,4 +473,13 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
 
   @override
   AtLookUp? get atLookUp => _atLookUp;
+
+  @override
+  AtChops? atChops;
+
+  @override
+  Future<bool> isOnboarded() async {
+    // #TODO implement once AtClient offline access feature is complete.
+    throw UnimplementedError();
```

Comment:
Can you add a link to the ticket for the offline access feature please?

Reply:
done